libgnucash/backend/xml/gnc-xml-backend.hpp

```diff
@@ -60,7 +60,7 @@ class GncXmlBackend : public QofBackend
     std::string m_dirname;
     std::string m_lockfile;
     std::string m_linkfile;
-    int m_lockfd;
+    int m_lockfd = -1;
```

Comment:
How about use a `constexpr int INVALID_FILE_HANDLE = -1` and use that to compare `m_lockfd`?

Reply:
That's an unnecessarily level of complexity for an invalid file descriptor...